Monika After Story/game/script-topics.rpy

```diff
@@ -67,6 +67,54 @@ label monika_god:
     m "So, from that perspective..."
     m 5a "I don't think it's too far-fetched for there to be a god - if Earth was nothing but his playset."
     return
+    
+init 5 python:
+    #This is a list of keywords for this topic
+    for key in ['hamlet', 'shakespeare']:
+        monika_topics.setdefault(key,[])
+        monika_topics[key].append('monika_hamlet') #id
+    monika_random_topics.append('monika_hamlet')
+
+label monika_hamlet:
+
+    menu:
+         m1d "[player], do you enjoy Shakespeare?"
+            "Yes.":
+                m 1k "Oh! Wonderful!"
+                m 1b "So, you know how I've had a lot of time recently?"
+                m 1j "Seeing as I'm immortal and time doesn't really pass in this place."
+                m 1b "Well...."
+                m "I've had time to practice and learn a couple of Shakespeare's more memorable speeches."
+                m 1m "Mind you, I'm not really the biggest fan of Shakespeare..."
+                m 1s "But I have to admit he's written some pretty soliloquies."
+                m 4k "Now... without further ado! Let me recite you a little something I memorized from Hamlet:"
+                m 1q "To be, or not to be--that is the question:"
+                m "Whether 'tis nobler in the mind to suffer the slings and arrows of outrageous fortune,"
+                m 1c "Or to take arms against a sea of troubles, And by opposing end them?"
+                m 1f "To die,"
+                m "To sleep, no more;"
+                m 1c "And by a sleep to say we end the heart-ache and the thousand natural shocks that flesh is heir to,"
+                m 1q "'Tis a consummation devoutly to be wish'd."
+                m "..."
+                m 5a "So? how did you like it?"
+                m 1j "I tried to be as expressive as possible for you, my love!"
```

Comment:
I would actually add a "..." between these two strings because I feel it isn't really a smooth enough transition currently.